### PR TITLE
Add vegnomes

### PR DESCRIPTION
**Pull request by Bread-VeSea**
Initial commit of VeGnomes NFT. Go live Saturday, 22nd

### CODE REVIEW

**Review by XJWX89 — CHANGES_REQUESTED** [review]
hi @Bread-VeSea , thanks for the submission ! currently the `manifest.json` on `main` branch ***does not*** contain `category ` & `contracts` fields. Moreover , please format the `manifest.json` before the submission.

**Comment by Bread-VeSea** [comment]
@XJWX89 - manifest fixed. I was trying to future proof the content but got sloppy - sorry about that.

Also having to correct it via mobile and it's not playing nice, but think I have it. Let me know if there are any issues.

File: apps/io.vesea.vegnomes/manifest.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# manifest.json
```
Comment (AsbertMa) [inline apps/io.vesea.vegnomes/manifest.json:1]: what's this？

**Comment by Bread-VeSea** [comment]
@AsbertMa - text removed and shame added to family name.

**Review by XJWX89 — APPROVED** [review]

**Pull request merged**